src/IRac.cpp

```diff
@@ -3056,6 +3067,8 @@ int16_t IRac::strToModel(const char *str, const int16_t def) {
     return lg_ac_remote_model_t::GE6711AR2853M;
   } else if (!strcasecmp(str, "AKB75215403")) {
     return lg_ac_remote_model_t::AKB75215403;
+  } else if (!strcasecmp(str, "AKB74955603")) {
```

Comment:
Use modeltostr function to remove duplicate string constant

Reply:
Worth a shot. I'll try it out. Not sure it will save us much, except only one canonical place for the string.
e.g. the call to `String irutils::modelToStr` plus args, plus a `.c_str()` at the end may be more space (bytes) than just duplicating the text.
e.g.
"AKB74955603" is approx 12 bytes of prog storage.
vs.
`modelToStr(protocol, model).c_str()` where protocol & model are 32 bits each (enums are 4 bytes), so, 8 bytes, and we haven't even gotten to the instructions for placing it on & off the stack, and the two function calls (JSRs) etc.
IMHO, I think it's gunna cost way more space to have a de-duped string. 
But .. I do like having only a single string to maintain. 🤷 

I remember struggling with this when I wrote it. :-/

Reply:
Yepp, gotcha.
Introduce \0 separated string same as protocols then?

Reply:
The protocols solution has the advantage that it only grows in one direction/dimension. The protocols/models is two dimensional, thus not easily _smooshable_.

Not saying can't be done or that I'm against it. I just don't know how, of the top of my head.
It's to small a problem (right now) to recover the expense of making it smaller/neater ... IMHO.

That said, if someone wants to implement something, go for it!

Reply:
I was thinking of a global models list that grows in only one dimension, and then convert between existing protocol models to/from global list.

New models could live in the global only list.
Downside being that disabling protocols would not reduce size of that list.

We should not stop this PR for it.

Reply:
Oooh! That's a good idea.

Agreed. Won't hold up this PR for it, but it's pending confirmation anyway.